docs/source/distributed.pipelining.rst

```diff
@@ -156,7 +156,7 @@ First off, let us instantiate a ``PipelineStage`` instance:
   # Pipeline stage is our main pipeline runtime. It takes in the pipe object,
   # the rank of this process, and the device.
   from torch.distributed.pipelining import PipelineStage
-  stage = PipelineStage(pipe, rank, device)
+  stage = TracerPipelineStage(pipe, rank, device)
```

Comment:
will wait for https://github.com/pytorch/pytorch/pull/128070 to land so this doesnt cause a merge conflict